python/arcticdb/version_store/processing.py

```diff
@@ -266,10 +266,16 @@ def value_list_from_args(*args):
 PythonProjectionClause = namedtuple("PythonProjectionClause", ["name", "expr"])
 PythonGroupByClause = namedtuple("PythonGroupByClause", ["name"])
 PythonAggregationClause = namedtuple("PythonAggregationClause", ["aggregations"])
-PythonRowRangeClause = namedtuple("PythonRowRangeClause", ["row_range_type", "n"])
 PythonDateRangeClause = namedtuple("PythonDateRangeClause", ["start", "end"])
 
 
+class PythonRowRangeClause(NamedTuple):
+    row_range_type: _RowRangeType = None
+    n: int = None
+    start: int = None
+    end: int = None
```

Comment:
Note to reviewers: using the `NamedTuple` class allows specifying default values to use either:

```python
PythonRowRangeClause(row_range_type=_RowRangeType.HEAD, n=n))
```
```python
PythonRowRangeClause(row_range_type=_RowRangeType.TAIL, n=n))
```
```python
PythonRowRangeClause(start=start, end=end))
```